src/main/java/com/google/sps/data/UserQuery.java

```diff
@@ -147,4 +152,17 @@ public UserQuery(String query, String tags, String authors) {
     }
     return new Query.CompositeFilter(Query.CompositeFilterOperator.OR, authorsFilters);
   }
+
+  /**
+   * Takes in a string which ha commas and returns a collection (currently an arrayList<string>)
+   * which contans all the substrings which were separated by commas. Can take a null parameter
+   * and empty string, and should return a collection with no elements in both cases. 
+   */
+  private ArrayList<String> convertToCollection(String params) {
```

Comment:
I was hesitant since the idea was eventually this would be converted to a set, so I didn't want to have to change the name everywhere, but since the internship is almost over and I don't have time, I'll go ahead and change it.